src/Avalonia.Visuals/Win32/AccentColor.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Avalonia.Media;
+
+namespace Avalonia.Win32
+{
+    internal static class AccentColor
```

Comment:
P/Invoke code isn't allowed in x-platform part of the framework unless with a few exceptions that we are slowly getting rid of. Move that to Avalonia.Win32.